benches/bench.rs

```diff
@@ -1,7 +1,7 @@
 use std::iter;
 
 use criterion::{criterion_group, criterion_main, Criterion};
-use event_listener::Event;
+use event_listener::{prelude::*, Event};
```

Comment:
I always find preludes annoying (except in case of std) since their use makes it hard to read the code (especially if there are multiple preludes being imported). I know you introduced them in e001c7a0 so it's not super relevant to this PR but I would like to know if you feel very strongly in favor of them. If not, I'd like to remove them.

Reply:
I added them mostly to make importing the at-the-time new `IntoNotification` trait easier. If you think they should be removed I'm fine with that.

Reply:
> I added them mostly to make importing the at-the-time new `IntoNotification` trait easier.

Yeah, I understand. They do make it easier to write but there is a cost to readability. Modern editors make it easier for your in most cases to import the needed traits anyway.

> If you think they should be removed I'm fine with that.

Yes, please.